src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+  <div id="amalgamating-businesses">
+    <v-btn
+      id="btn-add-amalgamating-business"
+      outlined
+      color="primary"
+      class="btn-outlined-primary"
+      :disabled="addAmalgatingBusinessDisabled"
+      @click="onAddBusinessClick()"
+    >
+      <v-icon>mdi-domain-plus</v-icon>
+      <span>Add an Amalgamating Business</span>
+    </v-btn>
+
+    <v-btn
+      v-if="isRoleStaff"
+      id="btn-add-amalgamating-business"
+      outlined
+      color="primary"
+      class="ml-2 btn-outlined-primary"
+      :disabled="addAmalgatingForeignBusinessDisabled"
+      @click="onAddForeignBusinessClick()"
+    >
+      <v-icon>mdi-domain-plus</v-icon>
+      <span>Add an Amalgamating Foreign Business</span>
+    </v-btn>
+
+    <!-- Add an Amalgamating Business button clicked -->
+    <v-expand-transition>
+      <v-card
+        v-if="isAddingAmalgamatingBusiness"
+        flat
+        class="section-container mt-4"
+      >
+        <v-row
+          no-gutters
+        >
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <label>Add a Business Registered in BC</label>
+          </v-col>
+
+          <v-col
+            cols="12"
+            sm="8"
+            class="ml-8"
+          >
+            <span>Enter the name or the incorporation number of the registered BC business
+              to add to this application.
+            </span>
+            <BusinessLookup
+              :showErrors="false"
+              :businessLookup="initialBusinessLookupObject"
+              :BusinessLookupServices="BusinessLookupServices"
+              label="Business Name or Incorporation Number"
+              @setBusiness="setAmalgamatingBusiness($event)"
+            />
+            <v-row
+              class="justify-end mr-0 mt-2"
```

Comment:
Moving the button to right as per the UXPin.

Reply:
OK but v-card has too much right whitespace.

![image](https://github.com/bcgov/business-create-ui/assets/10002889/312187f5-a13d-4b78-9a38-ef8e5536f190)